src/main/java/org/jabref/gui/Main.css

```diff
@@ -136,22 +323,22 @@
 }
 
 .menu-bar {
-    -fx-background-color: white;
+    -fx-background-color: -fx-body-color;
     -fx-background-insets: 0;
     -fx-background-radius: 0;
 }
 
-.menu-bar > .container > .menu-button > .label {
-    -fx-padding: 0.41777em 0.41777em 0.41777em 0.41777em; /* 5 5 5 5 */
-}
+/*.menu-bar > .container > .menu-button > .label {*/
```

Comment:
What's the reason for uncommenting these menu bar customization? If I remember correctly, they make the menu items a bit bigger to align with the MaterialDesing specifications.

Reply:
It doesn't look good. Even the default padding is larger than it is in other applications, but it looks OK. Compare these two images that show additionally the header of the mail program. First, as I have it now

![current](http://i.imgur.com/tZZfKhQ.png)

And here with the additional padding

![old](http://i.imgur.com/LvlXCde.png)

The menu items look lost in the menu bar. The spacing is far too large. I really would like to keep the default setting.

Btw, in the colorful version, we have now on `maintable-beta`, this kind of made sense because it tried to mimic the look of today's web-pages with their very large top menus. If you decide to work on a different scheme, the large padding might be an option again. But in this scheme, it doesn't work and I'd like to keep it simple and unsurprising for the user.

Reply:
I still like the second screenshot more and find it more touch-friendly. But, well, we just have different tastes and I definitely understand your viewpoint.
Probably every dev should create an own theme and we then vote which should be the default 😸